src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,72 @@
+import { ClassNames, css } from '@emotion/core';
+import * as React from 'react';
+import { Tab } from 'react-tabs';
+import { useTheme } from '../theme';
+
+const StyledTab: React.FunctionComponent<{ disabled?: boolean }> & { tabsRole: string } = props => {
+  const theme = useTheme();
+  const styles = tabStyles(theme.tabs);
+  return (
+    <ClassNames>
+      {({ css: getClassName }) => (
+        <Tab
+          {...props}
+          className={getClassName(styles.tabStyle)}
+          disabledClassName={getClassName(styles.disabledTabStyle)}
+          selectedClassName={getClassName(styles.selectedTabStyle)}
+        >
+          {props.children}
+        </Tab>
+      )}
+    </ClassNames>
+  );
+};
+
+StyledTab.tabsRole = 'Tab';
+
+const tabStyles = (tabTheme: { fg: string; bg: string }) => {
+  return {
```

Comment:
Could we use theme hook here?
I know it doesn't really matter, but we had such a convention in all components.

Reply:
Conventions are important! I'll fix!

Reply:
Done in 8bdac75